app/controllers/subscribers.js

```diff
@@ -108,10 +108,13 @@ export default Controller.extend(PaginationMixin, {
 
         sortByColumn(column) {
             let table = this.get('table');
+            let valuePath = column.get('valuePath');
+
+            valuePath = valuePath === 'createdAtUTC' ? 'created_at' : valuePath.trim().underscore();
```

Comment:
Not sure, how we could fix this... `valuePath` in `columns` is `createdAtUTC` now. That means, everytime `this.get('valuePath').trim().underscore()` was done, the property name changed to `created_at_utc` which, of course, is not correct for the server request.

I tried to work with an `alias`, but then the `columns` build would error... I couldn't figure out, if it's because of `ember-light-table` or if there's a better way to fix that... 

Happy for some input/ideas here! 😃